app/models/katello/root_repository.rb

```diff
@@ -455,6 +456,17 @@ def format_arches
       end
     end
 
+    def allow_updates?
+      # Allow updates for non-container-push repos and for updating exclusively the allow_updates flag
+      return true if allow_updates
+      return true if changed.size == 2 && will_save_change_to_allow_updates?
```

Comment:
I don't see this defined anywhere. Is this a Rails helper?

Reply:
Yes, that's a rails helper. I forget the exact syntax but would using changed.contains('allow_updates') be more readable? 

There's also a chance this will be moot if Ian's suggestion about allowing certain backend fields to be updated pans out.

Reply:
> would using changed.contains('allow_updates') be more readable?

I would've understood that on the first try :stuck_out_tongue:

Reply:
This no validation longer exists so I'll mark as resolved.